settings/provisioner/topology/ceph.yml

```diff
@@ -7,11 +7,13 @@ os:
     variant: "rhel7"
 disks:
     disk1: &disk1
+        import_url: null
```

Comment:
OK. We can leave it in.

Regards
Yair
On May 22, 2016 13:24, "Oleksii Baranov" <notifications@github.com> wrote:

> In settings/provisioner/topology/ceph.yml
> <https://github.com/rhosqeauto/InfraRed/pull/244#discussion_r64147651>:
>
> > @@ -7,11 +7,13 @@ os:
> >      variant: "rhel7"
> >  disks:
> >      disk1: &disk1
> > +        import_url: null
>
> No. We don't need to define. I've added that just to indicate that
> import_url can be defined.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rhosqeauto/InfraRed/pull/244/files/fa1b81d21de22b28d45cc341ada6f1e1a0643b15#r64147651>
>